tests/test_ecommerce_extraction.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import annotations
+
+from itemadapter import ItemAdapter
+
+from tests.utils_extraction import get_web_poet_response
+from zyte_test_websites.ecommerce.extraction import (
+    TestProductListPage,
+    TestProductNavigationPage,
+    TestProductPage,
+)
+
+
+async def test_product_extraction(ecommerce_client):
```

Comment:
Could you please also add a "negative" test, where product page object is called on a non-product page?